src/preamble.js

```diff
@@ -487,27 +487,36 @@ async function getWasmBinary(binaryFile) {
 
 #if SPLIT_MODULE
 {{{ makeModuleReceiveWithVar('loadSplitModule', undefined, 'instantiateSync') }}}
-var splitModuleProxyHandler = {
-  get(target, prop, receiver) {
-    return (...args) => {
+var wasmImportsProxyHandler = {
```

Comment:
Can we keep the splitModule prefix (or at least include it somewhere in the name) so its clear when this needed/used?

Reply:
Renamed it to `splitImportsProxyHandler`.